### ISSUE TITLE
[bug] ValueError: too many values to unpack (expected 2)

### ISSUE THREAD

**Issue by 7MinSec** [p0dalirius/Coercer#77]
Hello,

I'm on a pentest and using Coercer like usual:

````
coercer.py coerce -u 7ms -p 'pass' -t SOME.VICTIM.IP.ADDY -l LOCAL.KALI.IP.ADDY
````

When I do I get the error `ValueError: too many values to unpack (expected 2).`

After some troubleshooting I think it might be the password for my account.  Because if I run coercer without the `-p` and type it in when prompted for it, I still get the same error.  But if I type in something like `test` for the password when prompted for it, I get the (expected) error:

```
STATUS_LOGON_FAILURE - The attempted logon is invalid. This is either due to a bad username or authentication information.
[+] All done! Bye Bye!
```

I can't post the password for obvious reasons, but it's 25 characters long, has a mix of upper and lowercase characters and numbers, and ends with an asterisk.

**Comment by 7MinSec** [comment]
Ahhhh sorry for the spam but this may not be pw-related after all.  Investigating.

**Comment by brettgus** [comment]
Also ran into this issue.  Noticed it was fixed with issue #65.  Updated my install to the most recent version and confirmed that resolved it for me.

**Comment by 7MinSec** [comment]
Thanks for the update @brettgus brettgus, I was running the latest version as of the time this message was posted, BUT what I found out is that the environment I was in had the same internal/external domain name, and I was having sporadic auth issues for ALL Linux tools I was using.  Once I edited hosts file to point the domain to an internal domain controller, everything worked fine.

TLDR: it was a "me problem"  :-)
